features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -343,9 +343,11 @@
     <feature name="openhab-binding-ipp" description="IPP Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-mdns</feature>
-        <bundle>mvn:org.apache.httpcomponents/httpcore-osgi/4.2.3</bundle>
-        <bundle>mvn:org.apache.httpcomponents/httpclient-osgi/4.2.3</bundle>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.ipp/${project.version}</bundle>
+        <feature prerequisite="true">wrap</feature>
```

Comment:
Just a question: Is this `prerequisite` for `wrap` always needed when a jar is wrapped?

Reply:
Yes otherwise the Wrap URL protocol handler feature will not be installed and you'll see exceptions.
See https://github.com/openhab/openhab-core/pull/676.

Reply:
Ok. Then I think we need to add some more :smile: I missed at least one, and it looks like some more are missing.

Reply:
Preventing such issues and the duplication was the reason why I tried to get it added `openhab-runtime-base` :wink: